src/AppBundle/Repository/PostRepository.php

```diff
@@ -28,6 +28,14 @@
 class PostRepository extends EntityRepository
 {
     /**
+     * @return Post[]
+     */
+    public function findAll()
```

Comment:
What about to call the `findBy` method instead of the `findAll` in the controller and don't change the `findAll` behavior at all?